runtime/near-vm-errors/src/lib.rs

```diff
@@ -213,7 +213,6 @@ pub enum HostError {
     #[cfg(feature = "protocol_feature_alt_bn128")]
     AltBn128SerializationError { msg: String },
     /// General errors for ECDSA recover.
-    #[cfg(feature = "protocol_feature_math_extension")]
     ECRecoverError { msg: String },
```

Comment:
I believe this should be moved before the nightly features to ensure the correct union tag